fpga_interchange/testarch_generators/device_resources_builder.py

```diff
@@ -215,13 +216,15 @@ def __init__(self, ref, type):
 
 
 class PIP():
-    def __init__(self, wire0, wire1):
+    def __init__(self, wire0, wire1, delay_type):
         self.wire0 = wire0
         self.wire1 = wire1
 
         self.is_directional = True
-        self.is_buffered20 = True  # TODO:
-        self.is_buffered21 = True
+        self.is_buffered20 = False  #True  # TODO:
```

Comment:
Ideally, we'd test a mix of buffered and unbuffered PIPs. In practice, buffered PIPs do tend to be more common these days.

Reply:
I've added unbuffered PIPs and buffered PIPs to testarch Now PIP's that connect to sites are unbuffered, and interconnect ones are buffered

Reply:
Great, that sounds like a realistic mix